torch/nn/_intrinsic/quantized/modules/functional_relu.py

```diff
@@ -0,0 +1,17 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+from torch.nn._intrinsic import AddReLU as NNAddReLU
+
+class AddReLU(torch.nn.Module):
```

Comment:
Also, to reuse from_float, I think you'll need to inherit from `nnq.Add`?

Reply:
nnq.Add won't exist anymore -- we are depricating it in #23635